### PR TITLE
Show version number and release date

### LINKED ISSUE #1887: Show version number and release date

### PR DESCRIPTION
**Pull request by sorja**
Resolves #1887
Adds footer text to the right displaying build date and build version.

image

### CODE REVIEW

File: src/client/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -61,7 +64,11 @@ const Footer: React.FC = () => {
         </a>
       </div>
 
-      <span className="copyright">&copy; FAO, {new Date().getFullYear()}</span>
+      <div className="footer__version">
```
Comment (minotogna) [inline src/client/components/Footer/Footer.tsx:68]: Better to show before copyright then | Platform version vX.x.x-duddd-date

Comment (minotogna) [reply src/client/components/Footer/Footer.tsx:68]: > Better to show before copyright then | Platform version vX.x.x-duddd-date 

Or maybe even below copyright with smaller font size and lighter grey .

Comment (sorja) [reply src/client/components/Footer/Footer.tsx:68]: Styling updated, added new config in scss:
`$font-xxxs: 10px;` for smaller fonts

**Review by rfontanarosa — APPROVED** [review]

**Comment by minotogna** [comment]
can you update the screenshot ?

**Review by minotogna — APPROVED** [review]

**Pull request merged**